### DISCUSSION TITLE
Is it possible to hide/remove the ZX Simulator panel?

### DISCUSSION THREAD

**Discussion by remy** [maziac/DeZog#49] [Q&A] [answered]
I find myself needing the register view much more than the simulator view (where the screen and keyboard appear). I know now that I need to use `"commandsAfterLaunch": [ "-rmv" ]` to bring that back to life, but is it possible for me to configure to launch with the "Z80 Simulator" window closed?

(I don't think closed should be a default - but an opt-in)

**Comment by maziac** [comment, marked as answer]
You want to use zsim but without the window (webview)?
Interesting.

It's unlikely that this will happen. 
At the moment the zsim webview is only the display and the port input/output.
But while I implemented the audio output I found a major restriction in the design.
There is lot of communication required between the actual simulator code and the webview. 
I.e. a lot of unnecessary traffic.
And furthermore it is extremely difficult to keep the both in sync. For audio this was really a pain.

The idea is now, for a future DeZog 3.0, to move all of the simulator code into the webview.
I.e. I would need that window to do the simulation.

The one thing you can do, if that helps a bit, is turning most of the displayed things off, e.g. keyboard, display etc.
But the window itself will stay there.

I'm not sure. Could be that it is possible somehow to move the zsim window in the background if that would help?

**Reply by remy** [reply]
No, that's fine - I was more just checking that there wasn't a way already - I totally understand that there's a lot of moving parts in this software, and it's really not anywhere near a problem for me to either tab back to the register view or close it manually.

> It's unlikely that this will happen.

👍
